operations/operations/company-processes/publishing/publishing-guidelines/voice-tone-and-writing-style-guidelines/documentation-style-guide.md

```diff
@@ -1,80 +1,123 @@
 
-Write documents in English. Use American spelling, and avoid constructs where you are forced to write either _he_ or _she_, or _his_ or _her_. You can use _they_ or _their_ as singular forms instead.
+Avoid
+  3 cows ran for 6 kilometers when they saw 2300097 mosquitoes chasing them.
 
-### Numbers
+Preferred
+  Three cows ran for six kilometers when they saw 2,300,097 mosquitoes chasing them.
+
+[Helpful overview](https://www.grammarbook.com/numbers/numbers.asp):
 
-Spell out numbers when the number is the first word in a sentence or is less than or equal to ten \(i.e. one to nine\), otherwise use numeric digits \(i.e., 10 upwards\). Use commas to make long number strings easier to read.
+"In terms of numbering, policies and philosophies vary from medium to medium. America's two most influential style and usage guides have different approaches: The Associated Press Stylebook recommends spelling out the numbers zero through nine and using numerals thereafter—until one million is reached. Here are four examples of how to write numbers above 999,999 in AP style: 1 million; 20 million; 20,040,086; 2.7 trillion.
+
+The Chicago Manual of Style recommends spelling out the numbers zero through one hundred and using figures thereafter — except for whole numbers used in combination with hundred, thousand, hundred thousand, million, billion, and beyond (e.g., two hundred; twenty-eight thousand; three hundred thousand; one million). In Chicago style we would write four hundred, eight thousand, and twenty million with no numerals. Chicago style would require numerals for 401; 8,012; and 20,040,086."
 
 ### Commas
 
 We use the Oxford \(serial\) comma in our documentation.
 
-* For example: "Mattermost's writing style is clear, concise, and simple." as opposed to "Mattermost's writing style is clear, concise and simple."
+Avoid
+  The cows ran from wolves, coyotes and mosquitoes.
+
+Preferred
+  The cows ran from wolves, coyotes, and mosquitoes.
 
 ### Capitalization
 
-Use Title Case for document names/titles.
+#### Document titles
 
-* For example: "Deployment Guide for Organizations"
+We use Title Case for document names/titles. A document title is usually called an H1 heading and is either denoted by `#` in Markdown or `======` in reStructuredText. The title appears in the left-hand navigation and at the top of the table of contents. Use a title that accurately reflects the content of the document. People scan the table of contents looking for answers; it's often faster than using the built-in search engine.
 
-Use Sentence case for section titles and headings.
+**Use Title Case for document titles \(e.g. "This is an Article about Documentation"\). Where a word/title/name is lowercase, retain that casing in the title \(e.g. "This Article Covers the mmctl Tool"\).**
 
-* For example: "Writing guidelines for editors"
+Instead of *"Deployment guide for organizations"* we write *"Deployment Guide for Organizations"*
+
+#### Section titles and headings
```

Comment:
Let's consider covering gerund use here as well for H2 and lower headings in a future iteration of this page. Right now, we're using gerunds inconsistently, and providing clear guidance on our preferences will help contributors going forward.

Reply:
Oh that's a great idea, thank you! 

Something like "How to name pages" and then an explanation of gerunds, with examples of avoid and preferred.